io-package.json

```diff
@@ -192,7 +192,7 @@
                 "type": "string",
                 "read": true,
                 "write": false,
-                "role": "indicator"
```

Comment:
The type sould be number, what will be an appropriate role für a read only number?